src/model/SolarCollectorFlatPlatePhotovoltaicThermal.hpp

```diff
@@ -84,10 +87,11 @@ namespace model {
     /** @name Setters */
     //@{
 
-    /// Deletes the current parameters and clones the parameters passed in
-    bool setSolarCollectorPerformance(const SolarCollectorPerformancePhotovoltaicThermalSimple& parameters);
+    /// Deletes the current performance and clones the performance passed in
+    bool setSolarCollectorPerformance(const SolarCollectorPerformancePhotovoltaicThermalBIPVT& performance);
+    bool setSolarCollectorPerformance(const SolarCollectorPerformancePhotovoltaicThermalSimple& performance);
 
-    /// Deletes the current parameters and constructs a new default set of parameters
+    /// Deletes the current performance and constructs a new default set of performance
```

Comment:
Maybe I missed it, but where does it actually construct a new default set of performance?

Reply:
I see it now.